liquibase-core/src/main/java/liquibase/resource/ResourceAccessor.java

```diff
@@ -159,7 +158,12 @@ default SortedSet<String> list(String relativeTo, String path, boolean recursive
     default Resource getExisting(String path) throws IOException {
         Resource resource = get(path);
         if (!resource.exists()) {
-            throw new FileNotFoundException(FileUtil.getFileNotFoundMessage(path));
+            if (ChangeLogParserConfiguration.WARN_ON_MISSING_CHANGELOGS.getCurrentValue()) {
```

Comment:
Similarly to my previous comments, I would suggest extracting this logic as a function on this same interface.